README.md

```diff
@@ -10,9 +10,51 @@ Istio is an open-source service mesh that provides a uniform way to manage, conn
 
 The Istio module allows you to add Istio Operator to the Kyma runtime. Within Istio Operator, Istio Controller is responsible for installing, uninstalling, and managing Istio. For more information, read the [Istio Controller documentation](./docs/user/00-10-overview-istio-controller.md).
 
+## Install Istio Manager and Istio from the latest release
+
+### Prerequisites
+
+- Access to a Kubernetes (v1.24 or higher) cluster
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+### Procedure
+
+1. To install Istio, you must install latest Istio Manager and Istio CRD first. Run the following:
+
+   ```bash
+   kubectl apply -f https://github.com/kyma-project/istio/releases/latest/download/istio-manager.yaml
+   ```
+
+2. To get Istio installed, apply the latest default Istio CR:
+
+   ```bash
+   kubectl apply -f https://github.com/kyma-project/istio/releases/latest/download/istio-default-cr.yaml
+   ```
+
+   You should get a result similar to this example:
+
+   ```bash
+   istio.operator.kyma-project.io/default created
+   ```
+
+3. Check Istio CR state to verify a successful installation with:
+
+   ```bash
+   kubectl get -n kyma-system istios/default
+   ```
+
+   After successful installation you should get `Ready` state:
+
+   ```bash
+   NAME      STATE
+   default   Ready
+   ```
+
+For more installation options, visit [Install Istio Manager](./docs/contributor/01-00-installation.md)
```

Comment:
```suggestion
For more installation options, visit [Install the Istio module](./docs/contributor/01-00-installation.md).
```
Our installation doc is named 'Install the Istio module', but maybe the name 'Install Istio Operator' would be better. What do you think?